uber_sante/models/scheduler.py

```diff
@@ -4,48 +4,65 @@
 
 
 class RequestEnum(Enum):
-
     DAILY_REQUEST = 1
     MONTHLY_REQUEST = 2
 
 
+class BookingType(Enum):
+    ANNUAL = "ANNUAL"
+    WALKIN = "WALKIN"
+    ALL = "ALL"
```

Comment:
what do you mean by "ALL"?

Reply:
When you make a schedule_request you can request to query for annual, walkin or all